CMakeLists.txt

```diff
@@ -199,9 +200,5 @@ add_subdirectory(Descent3)
 add_subdirectory(tools)
 add_subdirectory(netcon)
 
-# For now we don't need to build the scripts under windows, so we'll only include
-# the directory when building for linux/osx. In the future we may want to to fix bugs, etc.
-if(UNIX)
-  add_subdirectory(netgames)
-  add_subdirectory(scripts)
-endif()
+add_subdirectory(netgames)
```

Comment:
Since netgames are now being built, can we add them to the netgames directory in Descent3\Release or Descent3\Debug to package them with artifacts?

Reply:
Sure.
I need to look at the CMake stuff, how best to it. Or someone points me to a good solution for it.

Reply:
It's already done for the TCP~IP.d3c file. You could use that as a template.

Reply:
Thanks for the pointer.
I was about to add something like `set_target_properties(${NETGAME_MODULE} PROPERTIES LIBRARY_OUTPUT_DIRECTORY "$<TARGET_FILE_DIR:Descent3>/netgames")` to every CMake file in the netgames subdirectories, before I noticed that @Jayman2000 already has a custom target defined in #297 that copies the files. Maybe we can land his changes before I try to solve this differently.

Reply:
Indeed, will try to get that merged today.